client/src/partials/reports_proposed/variation_exploitation/variation_exploitation.html

```diff
@@ -0,0 +1,46 @@
+<header data-header>
+  {{'VARIATION_EXPLOITATION.BUILD_VARIATION_EXPLOITATION' | translate}}
+</header>
+
+<main>
+  <div class="panel panel-default">
+    <div class="panel-heading">{{'VARIATION_EXPLOITATION.DOCUMENT_CONFIG' | translate}}</div>
+      <div class="panel-body">
+        <form>
+
+          <div class="form-group">
+            <label class="control-label col-xs-4 required">
+              {{ "VARIATION_EXPLOITATION.FISCAL_YEAR" | translate }}
+            </label>
+            <div>
+              <select class="form-bhima" ng-model="variationCtrl.fiscal_year_id" ng-options="fy.id as fy.fiscal_year_txt for fy in variationCtrl.fiscalYears.data" required>
```

Comment:
You probably should provide a default `<option>` element for the user as a placeholder.  Selecting in the controller as you did [here](https://github.com/IMA-WorldHealth/bhima/pull/837/files#diff-516a7d58993316077d75d5d7e37c182fR24) will fail if there are no fiscal years.